### PR TITLE
Chronograf GS updates

### PR DESCRIPTION
**Pull request by noramullen1**
@stevebang -- Here's my updated installation and getting started pages. Can you take a look and we can meet when you're ready to go over any changes? Thanks!

### CODE REVIEW

File: content/chronograf/v1.6/introduction/installation.md [hunk 1]
```diff
@@ -7,109 +7,50 @@ menu:
...
+
+For instructions on building from source, see Chronograf's [README](https://github.com/influxdata/chronograf/blob/master/README.md#from-source) on GitHub.
+
+## Connect Chronograf to your InfluxDB instance or InfluxDB Enterprise cluster
+
+1. Point your web browser to [http://localhost:8888](http://localhost:8888).
+2. Fill out the form  that appears with the following details:
+  * **Connection String**: Enter the host name or IP of the machine that InfluxDB is running on, and be sure to include InfluxDB's default port `8086`.
+  * **Connection Name**: Enter a name for your connection string.
+  * **Username_ and _Password**: These fields can remain blank unless you've [enabled authorization](influxdb/v1.6/administration/config/#auth-enabled-false) in InfluxDB.
```
Comment (sanderson) [inline content/chronograf/v1.6/introduction/installation.md]: I'm not sure that the underscores are necessary. Are these the actual names of the fields or are they trying to add emphasis to "and"? If adding emphasis, it should be:

```md
**Username _and_ Password**
```

File: content/chronograf/v1.6/introduction/installation.md [hunk 2]
```diff
@@ -7,109 +7,50 @@ menu:
...
+  * **Telegraf Database Name**: Optionally, enter a name for your Telegraf database. The default name is Telegraf.
+3. Click **Add Source**.
+
+## Connect Chronograf to Kapacitor
+
+1. In Chronograf, select the configuration icon in the sidebar menu, then select **Add Config** in the **Active Kapacitator** column.
```
Comment (sanderson) [inline content/chronograf/v1.6/introduction/installation.md]: You might describe the configuration icon here: (the wrench)

File: content/chronograf/v1.6/introduction/installation.md [hunk 3]
```diff
@@ -7,109 +7,50 @@ menu:
...
+2. In the **Kapacitor URL** field, enter the hostname or IP of the machine that Kapacitor is running on, and be sure to include Kapacitor's default port: `9092`.
```
Comment (sanderson) [inline content/chronograf/v1.6/introduction/installation.md]: I think this should be separated into two sentences.

```md
...the machine that Kapacitor is running on. Be sure to...
```

File: content/chronograf/v1.6/introduction/installation.md [hunk 4]
```diff
@@ -7,109 +7,50 @@ menu:
...
-```
-
-The next time you un-TAR a new version of Chronograf, simply change the `<version>` portion of the command string and specify your existing `chronograf-v1.db` datastore.
-
-## Configuration and Security
 
-See [Configuring Chronograf](/chronograf/latest/administration/configuration/) for a description of the available configuration options.
-See [Managing security](/chronograf/latest/administration/managing-security/) for TLS and OAuth 2.0 setup instructions.
+1. Choose the download link for your operating system.
+  * Note that if your download includes a TAR package, we recommend specifying a location for the underlying datastore, `chronograf-v1.db`, outside of the directory from which you start Chronograf. This allows you to preserve and reference your existing datastore, including configurations and dashboards, when you download future versions.
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: how does the user do this? should we add `-b` to th examples to start chronograf?

File: content/chronograf/v1.6/introduction/installation.md [hunk 5]
```diff
@@ -7,109 +7,50 @@ menu:
...
+2. Install Chronograf:
+  * MacOS: `tar zxvf chronograf-1.6.2_darwin_amd64.tar.gz`
+  * Ubuntu & Debian: `sudo dpkg -i chronograf_1.6.2_amd64.deb`
+  * RedHat and CentOS: `sudo yum localinstall chronograf-1.6.2.x86_64.rpm`
+3. Start Chronograf:
+  * MacOS: `tar zxvf chronograf-1.6.2_darwin_amd64.tar.gz`
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: this is the same as above. To install chronograf for Mac, we should probably add the `brew` instructions. if we want to use the tar instructions, we have to ask the user to `cd` to the folder and run `./chronograf`

File: content/chronograf/v1.6/introduction/installation.md [hunk 6]
```diff
@@ -7,109 +7,50 @@ menu:
...
+  * Ubuntu & Debian: `sudo dpkg -i chronograf_1.6.2_amd64.deb`
+
+For instructions on building from source, see Chronograf's [README](https://github.com/influxdata/chronograf/blob/master/README.md#from-source) on GitHub.
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: i don't think we need this IMO. not the right audience and might be confusing to new users.

File: content/chronograf/v1.6/introduction/installation.md [hunk 7]
```diff
@@ -7,109 +7,50 @@ menu:
...
+
+## Connect Chronograf to your InfluxDB instance or InfluxDB Enterprise cluster
+
+1. Point your web browser to [localhost:8888](http://localhost:8888).
+2. Fill out the form with the following details:
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: Maybe say you can most likely keep the defaults if you're running influx locally

File: content/chronograf/v1.6/introduction/installation.md [hunk 8]
```diff
@@ -7,109 +7,50 @@ menu:
...
+  * **Connection String**: Enter the hostname or IP of the machine that InfluxDB is running on, and be sure to include InfluxDB's default port `8086`.
+  * **Connection Name**: Enter a name for your connection string.
+  * **Username** and **Password**: These fields can remain blank unless you've [enabled authorization](influxdb/v1.6/administration/config/#auth-enabled-false) in InfluxDB.
+  * **Telegraf Database Name**: Optionally, enter a name for your Telegraf database. The default name is Telegraf.
+3. Click **Add Source**.
+
+## Connect Chronograf to Kapacitor
+
+1. In Chronograf, select the configuration (wrench) icon in the sidebar menu, then select **Add Config** in the **Active Kapacitator** column.
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: can we add an image for the wrench icon?

File: content/chronograf/v1.6/introduction/installation.md [hunk 9]
```diff
@@ -7,109 +7,50 @@ menu:
...
+
+## Connect Chronograf to your InfluxDB instance or InfluxDB Enterprise cluster
+
+1. Point your web browser to [localhost:8888](http://localhost:8888).
+2. Fill out the form with the following details:
+  * **Connection String**: Enter the hostname or IP of the machine that InfluxDB is running on, and be sure to include InfluxDB's default port `8086`.
+  * **Connection Name**: Enter a name for your connection string.
+  * **Username** and **Password**: These fields can remain blank unless you've [enabled authorization](influxdb/v1.6/administration/config/#auth-enabled-false) in InfluxDB.
+  * **Telegraf Database Name**: Optionally, enter a name for your Telegraf database. The default name is Telegraf.
+3. Click **Add Source**.
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: how do i verify it worked?

File: content/chronograf/v1.6/introduction/installation.md [hunk 10]
```diff
@@ -7,109 +7,50 @@ menu:
...
+
+## Connect Chronograf to Kapacitor
+
+1. In Chronograf, select the configuration (wrench) icon in the sidebar menu, then select **Add Config** in the **Active Kapacitator** column.
+2. In the **Kapacitor URL** field, enter the hostname or IP of the machine that Kapacitor is running on. Be sure to include Kapacitor's default port: `9092`.
+3. Enter a name for your connection.
+4. Leave the **Username** and **Password** fields blank unless you've specifically enabled authorization in Kapacitor.
+5. Click **Connect**.
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: how do i verify it worked?

File: content/chronograf/v1.6/introduction/installation.md [hunk 11]
```diff
@@ -7,109 +7,50 @@ menu:
...
+
+1. Point your web browser to [localhost:8888](http://localhost:8888).
+2. Fill out the form with the following details:
+  * **Connection String**: Enter the hostname or IP of the machine that InfluxDB is running on, and be sure to include InfluxDB's default port `8086`.
+  * **Connection Name**: Enter a name for your connection string.
+  * **Username** and **Password**: These fields can remain blank unless you've [enabled authorization](influxdb/v1.6/administration/config/#auth-enabled-false) in InfluxDB.
+  * **Telegraf Database Name**: Optionally, enter a name for your Telegraf database. The default name is Telegraf.
+3. Click **Add Source**.
+
+## Connect Chronograf to Kapacitor
```
Comment (russorat) [inline content/chronograf/v1.6/introduction/installation.md]: What does this enable? do i need to do this?

File: content/chronograf/v1.6/introduction/getting-started.md [hunk 12]
```diff
@@ -10,267 +10,19 @@ menu:
...
-```
-
-#### 2. Start InfluxDB
-There's no need to edit InfluxDB's default [configuration](https://docs.influxdata.com/influxdb/latest/administration/config/) for the purposes of this guide.
-Just start InfluxDB with:
-```
-sudo systemctl start influxdb
-```
-
-#### 3. Verify that InfluxDB is Running
```
Comment (stevebang) [inline content/chronograf/v1.6/introduction/getting-started.md]: Should be "Verify that InfluxDB is running" -- use sentence-style headings

File: content/chronograf/v1.6/introduction/getting-started.md [hunk 13]
```diff
@@ -10,267 +10,19 @@ menu:
...
-Chronograf and the other components of the TICK stack are supported on a large number of operating systems and hardware architectures.
-Check out the [downloads](https://www.influxdata.com/downloads/) page for links to the binaries of your choice.
-
-For a complete list of the installation requirements, see the [Installation](/chronograf/latest/introduction/installation/) page.
-
-## InfluxDB Setup
-
-[InfluxDB](https://docs.influxdata.com/influxdb/latest/) is the time series database that serves as the data storage component of the TICK stack.
-
-#### 1. Download and Install InfluxDB
```
Comment (stevebang) [inline content/chronograf/v1.6/introduction/getting-started.md]: Should be "Download and install InfluxDB" -- use sentence-style headings

File: content/chronograf/v1.6/introduction/getting-started.md [hunk 14]
```diff
@@ -10,267 +10,19 @@ menu:
...
-
-## Requirements
-
-This guide walks through getting set up on an Ubuntu 16.04 installation, and is applicable to most flavors of Linux.
-Chronograf and the other components of the TICK stack are supported on a large number of operating systems and hardware architectures.
-Check out the [downloads](https://www.influxdata.com/downloads/) page for links to the binaries of your choice.
-
-For a complete list of the installation requirements, see the [Installation](/chronograf/latest/introduction/installation/) page.
-
-## InfluxDB Setup
```
Comment (stevebang) [inline content/chronograf/v1.6/introduction/getting-started.md]: Should be "InfluxDB setup" -- use sentence-style headings

File: content/chronograf/v1.6/introduction/installation.md (hunk 5 above)
Comment (stevebang) [reply content/chronograf/v1.6/introduction/installation.md]: I agree with Russ -- for Mac OS X, use the Homebrew install -- this keeps it much simpler.  Also, use "Mac OS X" as the name, not "MacOS", which is no longer used.

**Comment by noramullen1** [comment]
Russ actually told me the exact opposite.

File: content/chronograf/v1.6/introduction/installation.md (hunk 6 above)
Comment (stevebang) [reply content/chronograf/v1.6/introduction/installation.md]: It's sufficient for our users to be told how to install binaries without needing to build from source. Those who want to build will typically know where to get what they need and how to do it.

File: content/chronograf/v1.6/introduction/installation.md (hunk 5 above)
Comment (sanderson) [reply content/chronograf/v1.6/introduction/installation.md]: It's the other way around. "Mac OS X" is no longer used. It's only "MacOS" or "macOS".

File: content/chronograf/v1.6/introduction/installation.md (hunk 8 above)
Comment (stevebang) [reply content/chronograf/v1.6/introduction/installation.md]: Change to "select **In the Chronograf navigation menu, click **Configuration** (wrench icon/image) in the navigation menu, then ...". Use "select" when a user selects from a number of options in a list or in a dropdown menu.

File: content/chronograf/v1.6/introduction/installation.md (hunk 5 above)
Comment (stevebang) [reply content/chronograf/v1.6/introduction/installation.md]: I was wrong -- the official name now is "macOS". We'll have to change that everywhere in the docs (eventually).

**Pull request merged**